src/main/scala/org/hammerlab/guacamole/commands/StructuralVariantCaller.scala

```diff
@@ -0,0 +1,129 @@
+package org.hammerlab.guacamole.commands
+
+import org.apache.spark.SparkContext
+import org.apache.spark.util.StatCounter
+import org.hammerlab.guacamole.{ DistributedUtil, SparkCommand, Common }
+import org.hammerlab.guacamole.reads.{ PairedRead, Read, MappedRead }
+import org.kohsuke.args4j.{ Option => Args4JOption }
+
+import scala.collection.mutable.ArrayBuffer
+
+/**
+ * Structural Variant caller
+ *
+ * This currently looks for regions with abnormally large insert sizes. This typically indicates
+ * that there's been a large deletion.
+ *
+ * The output is a text file of genomic ranges where there's some evidence of an SV.
+ */
+object StructuralVariant {
+
+  val MAX_INSERT_SIZE = 25000
+  val BLOCK_SIZE = 25
```

Comment:
Can you define/explain a 'block' somewhere. I read this initially thinking it had something to do with the BAM file.